### ISSUE TITLE
[Bug] Times in lists are improperly parsed and formatted

### ISSUE THREAD

**Issue by dyhork** [mattermost/mattermost-plugin-walltime#63]
We built and installed Walltime from master (29f710be3187d7dbe44ab348b18acc90131f47ee)

Looks like times written as part of lists get botched formatting.

```
- Today at 5pm PT
- Tomorrow at 4pm PT
- 3/30 at 5pm PT
```
becomes
- `Today at 5pm PT
- Tomorrow at 4pm PT` (_Wed, Mar 29 6:00 PM PDT - Thu, Mar 30 5:00 PM PDT_)
- `3/30 at 5pm PT` (_Thu, Mar 30 6:00 PM PDT_)

when it should be
- `Today at 5pm PT` (_Wed, Mar 29 6:00 PM PDT_)
- `Tomorrow at 4pm PT` (_Thu, Mar 30 5:00 PM PDT_)
- `3/30 at 5pm PT` (_Thu, Mar 30 6:00 PM PDT_)

Basically seems to be missing two ` characters
